docs/meetings/dev/2024-12-17.md

```diff
@@ -0,0 +1,35 @@
+# Weekly meeting
+
+When: 12/24/24, 21:00 UTC.
```

Comment:
17/12/24? Sorry for wrong side of pond convention

Reply:
lol I don't know what day it is thanks!